libcxx/CMakeLists.txt

```diff
@@ -279,7 +279,9 @@ endif()
 
 # Feature options -------------------------------------------------------------
 option(LIBCXX_ENABLE_EXCEPTIONS "Use exceptions." ON)
-option(LIBCXX_ENABLE_RTTI "Use run time type information." ON)
+option(LIBCXX_ENABLE_RTTI
+  "Use run time type information.
+   This option may only be set to OFF when LIBCXX_ENABLE_EXCEPTIONS=OFF." ON)
```

Comment:
Is this only a problem within libc++ or is there some fundamental reason we can't do this (i.e. is RTTI always required for EH)?

Reply:
See #66117 which @ldionne just created.